api/api.go

```diff
@@ -286,6 +302,10 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 
 		status := api.getAlertStatus(a.Fingerprint())
 
+		if !showSilenced && len(status.SilencedBy) != 0 {
```

Comment:
This reminds me that we need to add some indication in the UI if an alert is inhibited